src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -190,6 +190,9 @@ private function initializeSubscribers(): void
             if (\is_string($subscriber)) {
                 $subscriber = $this->container->get($subscriber);
             }
+
+            trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Passing subscribers is deprecated, use listeners instead');
```

Comment:
`Using Doctrine subscribers as services is deprecated, declare listeners instead.`